src/app/products/charges/edit-charge/edit-charge.component.ts

```diff
@@ -0,0 +1,124 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { ProductsService } from 'app/products/products.service';
+import { ActivatedRoute, Router } from '@angular/router';
+
+@Component({
+  selector: 'mifosx-edit-charge',
+  templateUrl: './edit-charge.component.html',
+  styleUrls: ['./edit-charge.component.scss']
+})
+export class EditChargeComponent implements OnInit {
+
+/** Selected Data. */
+chargeData: any;
+/** User form. */
+form: FormGroup;
```

Comment:
change name to editChargeForm

Reply:
Imoorted PipesModule in shared-module.ts